pyatv/support/mdns.py

```diff
@@ -22,36 +22,41 @@
 logging.addLevelName(TRAFFIC_LEVEL, "Traffic")
 
 
-class QueryType(enum.IntEnum):
```

Comment:
The diff for this is not great, but `QueryType` and the DNS named tuples were moved further down in the file.